tox.ini

```diff
@@ -0,0 +1,10 @@
+[tox]
+envlist = py27
+
+[testenv]
+deps =
+  Testify
```

Comment:
despite our github calling it Testify, the actual package name is [testify](https://github.com/Yelp/Testify/blob/ab5eba82cacd91a2774bec955852eda5b3128984/setup.py#L4), p4